research/src/pages/documentation/get-involved.mdx

```diff
@@ -19,20 +19,20 @@ The best place to start is by participating in one or more of our communication
 
 ## Standardizing an Open UI Web Component
 
-The following guides walks you through the process of bringing a Component from idea to either a standardized Open UI Web Component and/or an implemented feature in a browser.
+The following guide walks you through the process of bringing a Component from idea to either, a standardized Open UI Web Component and/or, an implemented feature in a browser.
```

Comment:
agreed with guides --> guide, but not sure if the added commas make sense?

we could probably also remove “either”, as we already have “and/or” ?